lib/middleman-hashicorp/version.rb

```diff
@@ -1,5 +1,5 @@
 module Middleman
   module HashiCorp
-    VERSION = "0.3.38"
+    VERSION = "0.3.40"
```

Comment:
I think it makes sense to just bump to `0.3.39` as part of this PR?

Reply:
@jfreda Do you mean do a second PR for the pre-emptive bump to 0.3.40? Or don't do a pre-emptive bump at all?

Reply:
I think you could just push a version bump after this is merged (no PR needed).

Reply:
👍🏼